Mailchimp Interest Group dest. initial commit

mohan-h opened the pull request:
Initial commit

mohan-h committed:
Mailchimp Interest Group dest. initial commit

Initial commit

mohan-h committed:
Image paths corrected

mohan-h committed:
Updated mappings and guardrails

mapping : Elaborated Address field limitations
guardrails : 60 group name limit

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) endpoint. You can **add new contacts** or **update the information of existing [!DNL Mailchimp] contacts**, then **add or remove them from their desired groups** with ...
+
+[!DNL Mailchimp Interest Group] uses OAuth 2 with Authorization Code to communicate with the [!DNL Account Engagement] API. Instructions to authenticate to your [!DNL Mailchimp Interest Group] instance are further below, in the [Authenticate to destination](#authenticate) section.
```
[!DNL Account Engagement] API

^^ is this a leftover from Salesforce?

mohan-h replied:
Excellent spot, my apologies have corrected.

vgiurgiu replied:
No worries, we're both probably dreaming Account Engagement at this point 🙂

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+---
+title: Mailchimp Interest Group
```
quick reminder to edit title and naming across the page to account for the naming I suggested in https://launch-developers.slack.com/archives/C02Q3542CUU/p1682112194176349?thread_ts=1681135747.033639&cid=C02Q3542CUU

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+[[!DNL Mailchimp]](https://mailchimp.com) *(also known as [!DNL Intuit Mailchimp])* is a popular marketing automation platform and email marketing service used by businesses to manage and talk to contacts *(clients, customers, or other interested parties)* using mailing lists and email marketing campaigns.
+
+[!DNL Mailchimp Interest Group] uses [audiences](https://mailchimp.com/help/getting-started-audience/) and [groups](https://mailchimp.com/help/getting-started-with-groups/). Groups can be used to help organize your contacts within [!DNL Mailchimp] based on their interests and preferences and then broadcast group specific campaign emails.
```
can we briefly talk about interest categories in this section as well, please? -- https://mailchimp.com/developer/marketing/api/interest-categories/

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+Compared to [!DNL Mailchimp Tags] which you would use for internal classification, [!DNL Mailchimp Interest Group] is meant to manage subscriptions to topics of interest that your contacts might be interested in. *Note, Experience Platform also has a connection for [!DNL Mailchimp Tags], you can check it out on the [[!DNL Mailchimp Tags]](/help/destinations/catalog/email-marketing/mailchimp-tags. ...
+-->
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) endpoint. You can **add new contacts** or **update the information of existing [!DNL Mailchimp] contacts**, then **add or remove them from their desired groups** with ...
```
what about interest categories? how does that fit in here?

I'm thinking here about the output in Mailchimp described in lines 191 -193 https://github.com/AdobeDocs/experience-platform.en/pull/202/files#diff-8ea24d629e65b7ecb9e4674bacf11241a84d7a33d432bab1f1dc8440a06f2682R191-R193

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The sales department of a sporting good website wants to broadcast an email based marketing campaign to a list of contacts who have self identified themselves as being interested in soccer. The lists of contacts are segregated as batches in the data export received from the development team of the website and therefore need to be tracked. The team identifies an existing [!DNL Mailchimp] audience  ...
```
various grammar updates required, mostly about missing commas:

sporting good**s**

email-based

self-identified

segment's -> segments

After sending these segments to [!DNL Mailchimp Interest Groups], if any contacts do not exist in the selected [!DNL Mailchimp] audience, they get added to a group with the segment name that the contact belongs to.

If any contacts already exist in the [!DNL Mailchimp] audience or group, then their information is updated.

Once the data is sent over to [!DNL Mailchimp Interest Group], the Sales team can select and send the marketing campaign email to the soccer interest group within the [!DNL Mailchimp] audience.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+#### You need to have a [!DNL Mailchimp] account {#prerequisites-account}
+
+Before you can create a [!DNL Mailchimp Interest Group] destination, you must first ensure that you have a [!DNL Mailchimp] account. If you do not have one already visit the [[!DNL Mailchimp] signup page](https://login.mailchimp.com/signup/) to register and create your account.
```
comma:

If you do not have one already, visit the [[!DNL Mailchimp] signup page](https://login.mailchimp.com/signup/) to register and create your account.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+{style="table-layout:auto"}
+
+#### Identify [!DNL Mailchimp] data center {#identify-data-center}
```
data center subdomain, as discussed in the sync meeting and as seen in the Mailchimp documentation:
https://mailchimp.com/developer/marketing/docs/fundamentals/#api-structure

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+Next you need to identify your Mailchimp domain. To do this, log in to your [!DNL Mailchimp] account and check the URL in your browser.
+You’ll notice a value similar to *`us19`.admin.mailchimp.com*, Write down the prefix (`us19` in this example). You will need this prefix when connecting to the destination. 
+If you require guidance, refer to the [Mailchimp documentation](https://mailchimp.com/developer/marketing/guides/quick-start/#make-your-first-api-call).
```
we could also link to this page which describes the Mailchimp data center subdomain: 
https://mailchimp.com/developer/marketing/docs/fundamentals/#api-structure

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+Each of your [!DNL Mailchimp] audiences can contain up to 60 group names in a single group or across several groups within the same audience. Refer to [!DNL Mailchimp] [groups](https://mailchimp.com/help/getting-started-with-groups/) for any clarifications required. When you reach this limit you will get a `400 BAD_REQUEST Cannot have more than 60 interests per list (Across all categories)` messa ...
+
+Additionally refer to the [!DNL Mailchimp] [rate limits](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-limits) for detailed information about the limits imposed by the [!DNL Mailchimp] API.
```
Minor, missing comma:

Additionally, refer to the [...]

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**. You will be redirected to the [!DNL Mailchimp] authorization page.
+
+![Platform UI screenshot showing how to authenticate to Mailchimp.](../../assets/catalog/email-marketing/mailchimp-interest-group/authenticate-destination.png)
```
nitpick, but can we resize this and the next image to be smaller in size?

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/mailchimp-interest-group/destination-details.png)
+
+| Field | Description |
```
reminder to use the field names suggested in https://launch-developers.slack.com/archives/C02Q3542CUU/p1682112194176349?thread_ts=1681135747.033639&cid=C02Q3542CUU

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+    |`xdm: person.name.lastName`|`Attribute: LNAME`| |
+    |`xdm: person.birthDayAndMonth`|`Attribute: BIRTHDAY`| |
+
+    Additionally, `ADDRESS` is a special target field known as a `merge field` within your [!DNL Mailchimp] audience. The [[!DNL Mailchimp] documentation](https://mailchimp.com/developer/marketing/docs/merge-fields/) defines the required keys as `addr1`, `city`, `state`, and `zip`, and the optional keys `addr2` and `country`. The values for these fields must be strings. If any of the `ADDRESS` fi ...
```
minor, missing comma

If any of the `ADDRESS` field mappings are present, the destination passes the `ADDRESS` object [...]

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+    | `xdm: workAddress.postalCode` | `Attribute: ADDRESS.zip` |
+    | `xdm: workAddress.country` | `Attribute: ADDRESS.country` |
+
+    For example, you want to update the value for `country` with the contact's existing address field `addr1`, `city`, `state`, and `zip` values as `132, My Street, Kingston`, `New York`, `New York` and `12401`. To update the `country` you need to pass the existing values with changes *(if any)* and the new value for country. So the values in your dataset should be `132, My Street, Kingston`, `Ne ...
```
minor, missing comma

To reiterate, if you only pass [...]

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+* Log in to your [[!DNL Mailchimp]](https://login.mailchimp.com/) account. Then navigate to the **[!DNL Audience]** page. Next expand the **[!DNL Manage Contacts]** menu and select **[!DNL Groups]**.
```
missing commas:

Then, navigate to the **[!DNL Audience]** page. Next, expand the **[!DNL Manage Contacts]** menu and select **[!DNL Groups]**.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-group.md:
```diff
@@ -0,0 +1,213 @@
+
+![Mailchimp UI screenshot showing the Audience group page.](../../assets/catalog/email-marketing/mailchimp-interest-group/audience-groups.png)
+
+* Select the Group and check if the selected segments are created as categories sharing the segment name from Platform followed by an auto generated suffix. 
```
automatically generated suffix

why is the automatically generated suffix appended at the end?

mohan-h replied:
This destination uses the selected segment's names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and select the same segments, because the selected segment's names already exist in [!DNL Mailchimp] the create API call will fail. By adding the `auto generated suffix` this destination overcomes the limitation.

mohan-h replied:
Have also included it in the write up.

vgiurgiu replied:
Thanks, I left a minor follow-up comment on this section.

vgiurgiu reviewed (commented):
Thanks for creating this PR. I left some comments to be addressed before we can publish the destination documentation. let me know please if you have any questions, thanks!

mohan-h committed:
Connector name refactoring

Connector name changed, Paths refactored

mohan-h committed:
Review correction - API name

mohan-h committed:
Resizing images

mohan-h committed:
New image as field names have changed

mohan-h committed:
Addressing review comments

Changed field names as per guidance and addressed review comments

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,215 @@
+![Mailchimp UI screenshot showing the Audience group page.](../../assets/catalog/email-marketing/mailchimp-interest-categories/audience-groups.png)
+
+* Select the Group and check if the selected segments are created as categories sharing the segment name from Platform followed by an auto generated suffix.
+  * This destination uses the selected segment's names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and select the same segments, because the selected segment's names already exist in [!DNL Mailchimp] the create API call will  ...
```
Minor wording revision: 

This destination uses the selected segments' names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp] the destination adds a suffix to distinguish between the segments.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,215 @@
+
+![Mailchimp UI screenshot showing the Audience group page.](../../assets/catalog/email-marketing/mailchimp-interest-categories/audience-groups.png)
+
+* Select the Group and check if the selected segments are created as categories sharing the segment name from Platform followed by an auto generated suffix.
```
Select the Group and check if the selected segments are created as categories with the segment name from Platform, which may be followed by an automatically generated suffix.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,215 @@
+
+[[!DNL Mailchimp]](https://mailchimp.com) *(also known as [!DNL Intuit Mailchimp])* is a popular marketing automation platform and email marketing service used by businesses to manage and talk to contacts *(clients, customers, or other interested parties)* using mailing lists and email marketing campaigns.
+
+[!DNL Mailchimp Interest Categories] uses [audiences](https://mailchimp.com/help/getting-started-audience/), [groups](https://mailchimp.com/help/getting-started-with-groups/) and interest categories *(also known as group names or group titles)*. Each [!DNL Mailchimp] group is a list of interest categories. Contacts are associated with an interest category when they subscribe to one or more intere ...
```
Great context, I very much appreciate this section! Just one quick comment about a comma and minor wording revision:

Within an audience, you can also organize the contacts into groups and associate them with interest categories, and these can then be used to create segments. You can use these segments to broadcast targeted campaign emails to the subscribed contacts.

vgiurgiu reviewed (commented):
Just a few additional minor comments. Thanks for the quick updates!

mohan-h committed:
Updates against review comments

mohan-h committed:
Authentication changed from Oauth2 to Basic

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+---
+title: Mailchimp Interest Categories
+description: The Mailchimp Interest Categories destination allows you to export your contacts data and activate it within Mailchimp. The contacts can be managed as segments based on their interest category subscriptions.
```
Let's rewrite this in the spirit of customer-focused documentation. What can _you_ achieve with this destination? So my suggestion is around something like: 

Use the Mailchimp Interest Categories destination to export your contacts data and activate it within Mailchimp. Manage the contacts in Mailchimp as segments, based on their interest category subscriptions.

^^ please confirm that the above is still correct after my rewording.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+Compared to [!DNL Mailchimp Tags] which you would use for internal classification, [!DNL Mailchimp Interest Categories] is meant to manage subscriptions to topics of interest that your contacts might be interested in. *Note, Experience Platform also has a connection for [!DNL Mailchimp Tags], you can check it out on the [[!DNL Mailchimp Tags]](/help/destinations/catalog/email-marketing/mailchimp- ...
+-->
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) API to create [interest categories](https://mailchimp.com/developer/marketing/api/interest-categories/)  then add contacts from each of the selected Platform segments ...
```
slight rewording, adding the word 'and': 

This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) API to create [interest categories](https://mailchimp.com/developer/marketing/api/interest-categories/) and then add contacts from each of the selected Platform segments into a corresponding interest category.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Mailchimp batch subscribe or unsubscribe API]](https://mailchimp.com/developer/marketing/api/lists/batch-subscribe-or-unsubscribe/) API to create [interest categories](https://mailchimp.com/developer/marketing/api/interest-categories/)  then add contacts from each of the selected Platform segments ...
+
+[!DNL Mailchimp Interest Group] uses Basic Authorization to communicate with the [!DNL Mailchimp] API. Instructions to authenticate to your [!DNL Mailchimp Interest Categories] instance are further below, in the [Authenticate to destination](#authenticate) section.
```
probably not relevant to the end user that the destination uses Basic Authorization

mohan-h replied:
Ok, have nuked the entire paragraph.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+#### Gather [!DNL Mailchimp] API Key {#gather-credentials}
+
+You will need your **API Key** to authenticate the [!DNL Mailchimp Interest Categories] destination against your [!DNL Mailchimp] account. 
```
Let's also call out in clear here that the Mailchimp API Key serves as the password when connecting to the destination

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+#### Identify [!DNL Mailchimp] data center subdomain {#identify-data-center-subdomain}
+
+Next you need to identify your Mailchimp data center subdomain. To do this, log in to your [!DNL Mailchimp] account and navigate to the **API keys section** of your account. 
```
comma after Next

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+Write down the data center subdomain value *(`us14` in this example)*, you will need this value when you [fill in destination details](#destination-details).
+
+If you require guidance, refer to the [[!DNL Mailchimp] Fundamentals documentation](https://mailchimp.com/developer/marketing/docs/fundamentals/#api-structure).
```
Minor revision:

If you require further guidance

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+### Guardrails {#guardrails}
+
+Each of your [!DNL Mailchimp] audiences can contain up to 60 group names (or interest categories) in a single group or across several groups within the same audience. Refer to [!DNL Mailchimp] [groups](https://mailchimp.com/help/getting-started-with-groups/) for any clarifications required. When you reach this limit you will get a `400 BAD_REQUEST Cannot have more than 60 interests per list (Acro ...
```
comma after limit

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]**, search for [!DNL Mailchimp Interest Categories]. Alternatively you can locate it under the **[!UICONTROL Email marketing]** category.
```
comma after Alternatively

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+| [!DNL Mailchimp] destination | [!DNL Mailchimp] |
+| --- | --- |
+| **[!UICONTROL Username]** | Your [!DNL Salesforce Marketing Cloud] domain prefix. <br>For example if your domain is <br> *`mcq4jrssqdlyc4lph19nnqgzzs84`.login.exacttarget.com*, <br> you need to provide `mcq4jrssqdlyc4lph19nnqgzzs84` as the value. |
```
1. Comma after for example
2. sight wording update: you need to provide `mcq4jrssqdlyc4lph19nnqgzzs84` as the username value.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+| [!DNL Mailchimp] destination | [!DNL Mailchimp] |
+| --- | --- |
+| **[!UICONTROL Username]** | Your [!DNL Salesforce Marketing Cloud] domain prefix. <br>For example if your domain is <br> *`mcq4jrssqdlyc4lph19nnqgzzs84`.login.exacttarget.com*, <br> you need to provide `mcq4jrssqdlyc4lph19nnqgzzs84` as the value. |
+| **[!UICONTROL Password]** | Your [!DNL Mailchimp] **API Key**. <br> This is the API Key you had noted down in the [Gather [!DNL Mailchimp] credentials](#gather-credentials) section is of the form *key-dc*. <br>You can provide either the *key* portion or the entire form.<br> For example if your API Key is <br>*`0123456789abcdef0123456789abcde-us14`*,<br> you could provide either *`0123456789abcd ...
```
> This is the API Key you had noted down in the [Gather [!DNL Mailchimp] credentials](#gather-credentials) section is of the form *key-dc*.

Please revise the sentence above as it does not make grammatical sense.

Also, comma after **for example**

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/mailchimp-interest-categories/destination-details.png)
```
the screenshot shows Mailchimp Interest Tags Test. Let's please revise to the name of this destination.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+{style="table-layout:auto"}
+
+If the API Key you provided in the **[!UICONTROL Password]** field or the **[!UICONTROL Data Center Subdomain]** value are incorrect the UI displays a [!DNL Mailchimp] API error response: *`No options are available. Please verify the values selected for the following dependent fields: apiServer`* as shown below and you will not be able to select a value from the **[!UICONTROL Audience Name (Pleas ...
```
let's highlight this section as a TIP note.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+![Platform UI screenshot showing error if your Mailchimp API Key or Data Center Subdomain values are incorrect.](../../assets/catalog/email-marketing/mailchimp-interest-categories/error.png)
+
+To fix this error and proceed to the next step you need to provide the correct values. Refer to the [Identify [!DNL Mailchimp] data center subdomain](#identify-data-center-subdomain) and 
```
comma after next step

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select attributes]** category and select from the list of attributes populated from the [!DNL Mailchimp] API. *Note any custom attributes that have been added to the selected [!DNL Mailchimp] Audience will also be available.*
```
wording suggestion:

*Note any custom attributes that have been added to the selected [!DNL Mailchimp] Audience will also be available.* -> *Note that any custom attributes that you have added to the selected [!DNL Mailchimp] Audience will also be available for selection as target fields.*

let me know if my rewording is true to reality.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+    |`xdm: person.name.lastName`|`Attribute: LNAME`| |
+    |`xdm: person.birthDayAndMonth`|`Attribute: BIRTHDAY`| |
+
+    Additionally, `ADDRESS` is a special target field known as a `merge field` within your [!DNL Mailchimp] audience. The [[!DNL Mailchimp] documentation](https://mailchimp.com/developer/marketing/docs/merge-fields/) defines the required keys as `addr1`, `city`, `state`, and `zip`, and the optional keys `addr2` and `country`. The values for these fields must be strings. If any of the `ADDRESS` fi ...
```
what does this section mean for me a a user, though? is there any recommendation included here?

vgiurgiu replied:
please ignore, this is well explained by the example in the next section 👍

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select attributes]** category and select from the list of attributes populated from the [!DNL Mailchimp] API. *Note any custom attributes that have been added to the selected [!DNL Mailchimp] Audience will also be available.*
+
+    The default mappings available between your XDM profile schema and [!DNL Mailchimp Interest Categories] will be as below:
```
what does this mean? do you need to map `xdm: person.name.firstName` to `Attribute: FNAME`?

mohan-h replied:
Yes, I slightly reworded now to remove the word _default_.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+    For example, you want to update the value for `country` with the contact's existing address field `addr1`, `city`, `state`, and `zip` values as `132, My Street, Kingston`, `New York`, `New York` and `12401`. To update the `country` you need to pass the existing values with changes *(if any)* and the new value for country. So the values in your dataset should be `132, My Street, Kingston`, `Ne ...
+
+    An example with the completed mappings is shown below:
+    ![Platform UI screenshot example showing field mappings.](../../assets/catalog/email-marketing/mailchimp-interest-categories/mappings.png)
```
looks like in the screenshot the target field value for first name is incorrect and should be FNAME

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+![Mailchimp UI screenshot showing the Audience group page.](../../assets/catalog/email-marketing/mailchimp-interest-categories/audience-groups.png)
+
+* Select the Group and check if the selected segments are created as categories with the segment name from Platform, which may be followed by an automatically generated suffix.
+  * This destination uses the selected segments' names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp] the destination adds a  ...
```
comma missing:

If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp], the destination adds a suffix to distinguish between the segments.

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+* Select the Group and check if the selected segments are created as categories with the segment name from Platform, which may be followed by an automatically generated suffix.
+  * This destination uses the selected segments' names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp] the destination adds a  ...
+* Contacts whose emails did not exist in the group will be added to the newly created category.
```
can we change to present tense please?

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+* Select the Group and check if the selected segments are created as categories with the segment name from Platform, which may be followed by an automatically generated suffix.
+  * This destination uses the selected segments' names to create the interest category by leveraging the [[!DNL Mailchimp] Add interest category API](https://mailchimp.com/developer/marketing/api/interest-categories/add-interest-category/). If you create a new destination and activate the same segments, because the selected segments' names already exist in [!DNL Mailchimp] the destination adds a  ...
+* Contacts whose emails did not exist in the group will be added to the newly created category.
+* For contacts that already exist within the group the attribute field data will be updated. The contact is also added to the newly created category.
```
please change to present tense if possible. 
also, missing comma: 

For contacts that already exist within the group, the attribute field data [...]

vgiurgiu commented on help/destinations/catalog/email-marketing/mailchimp-interest-categories.md:
```diff
@@ -0,0 +1,239 @@
+
+### Error encountered if [!DNL Mailchimp] API Key or Data Center Subdomain values are incorrect {#incorrect-credentials-error}
+
+If the API Key you provided in the **[!UICONTROL Password]** field or the **[!UICONTROL Data Center Subdomain]** value are incorrect the UI displays a [!DNL Mailchimp] API error response: *`No options are available. Please verify the values selected for the following dependent fields: apiServer`* as shown below and you will not be able to select a value from the **[!UICONTROL Audience Name (Pleas ...
```
missing comma: 

value are incorrect, the UI displays

vgiurgiu reviewed (commented):
some comments, mostly around slight grammar suggestions and revisions. as always, please let me know if you have questions.

vgiurgiu reviewed (commented):
some comments, mostly around slight grammar suggestions and revisions. as always, please let me know if you have questions.

mohan-h committed:
Updates addressing review comments

mohan-h committed:
Explicitly calling out API Key as password

mohan-h committed:
Label changes

Data Center Subdomain to Data center, API Key to API key, Images updated and minor updates to description

mohan-h committed:
Error message updated

Error message updated, now refers to dataCenter instead of apiServer

vgiurgiu committed:
added last-substantial-update metadata in prep for release
